components/avatar.html

```diff
@@ -0,0 +1,118 @@
+                <div class="image_avatar">
+                    <img class="avatar_square " src="/images/avatar2.jpg " alt="">
+                    <img class="avatar_square " src="/images/avatar2.jpg " alt="">
+                    <img class="avatar_square " src="/images/avatar2.jpg " alt="">
+                </div>
+                <div class="carbon-img1"> <iframe
+                    src="https://carbon.now.sh/embed?bg=rgba%2831%2C129%2C109%2C1%29&t=night-owl&wt=none&l=auto&ds=false&dsyoff=20px&dsblur=68px&wc=false&wa=false&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=143%25&si=false&es=2x&wm=false&code=%2520%2520%2520%2520%2520%2520%2520%253Cimg%2520class%253D%2522avatar_square%2520%2522%2520src%253D%2522%252Fimages%252Favatar2.jpg%2520%2522%2520alt%253D%2522%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cimg%2520class%253D%2522avatar_square%2520%2522%2520src%253D%2522%252Fimages%252Favatar2.jpg%2520%2522%2520alt%253D%2522%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cimg%2520class%253D%2522avatar_square%2520%2522%2520src%253D%2522%252Fimages%252Favatar2.jpg%2520%2522%2520alt%253D%2522%2522%253E"
+                    style="width: 758px; height: 216px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                  </iframe></div>
+                
+
+                <h3  class="pad">Letter Avatar</h3>
+                <div class="image_avatar">
+                    <div class="avatar_letter av_blue ">A</div>
+                    <div class="avatar_letter av_pink">B</div>
+                    <div class="avatar_letter av_green">C</div>
+                </div>
+                <div class="carbon-img1"><iframe
+                    src="https://carbon.now.sh/embed?bg=rgba%2831%2C129%2C109%2C1%29&t=night-owl&wt=none&l=auto&ds=false&dsyoff=20px&dsblur=68px&wc=false&wa=false&pv=74px&ph=72px&ln=false&fl=1&fm=dm&fs=14px&lh=143%25&si=false&es=2x&wm=false&code=%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522avatar_letter%2520av_blue%2520%2522%253EA%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522avatar_letter%2520av_pink%2522%253EB%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522avatar_letter%2520av_green%2522%253EC%253C%252Fdiv%253E"
+                    style="width: 737px; height: 252px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                  </iframe></div>
+
+                <h3  class="pad">Size Avatar</h3>
+                <div class="image_avatar">
+                    <img class="avatar_size " src="/images/avatar2.jpg " alt="">
+                    <img class="avatar_size size_inc" src="/images/avatar2.jpg " alt="">
+                    <img class="avatar_size size_inc1" src="/images/avatar2.jpg " alt="">
```

Comment:
The sizes of the avatar looks similar. Differentiate one size from another so that it can be used for different purposes later. You have a slight size difference as of now which looks similar to the naked eye and cannot be used for different purposes or at larger scales.

Reply:
Okay! Anjali I will makes changes in size.